File: easy/pro-vowel-counter/VowelCounter.java
import java.util.Scanner;

public class VowelCounter {
    public static void main(String[] args) {
        
        long vowels = new Scanner(System.in)
            .nextLine()
            .chars()
            .filter(c -> "AEIOUaeiou".indexOf(c) != -1)
            .count();
        
        System.out.println(vowels);
    }
}

File: easy/pro-vowel-counter/vowel_counter.c
#include <stdio.h>

int main() {
    int vowels = 0;
    
    for (char c; scanf("%c", &c) == 1;) 
        for (int i = 0; i < 10; ++i) 
            if ("AEIOUaeiou"[i] == c) {++vowels; break;}
            
    printf("%d", vowels);
    return 0;
}
